make/Makefile-local-install

```diff
@@ -0,0 +1,80 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# ~~~~~~~~~~~~~~~~ LOCAL DIST INSTALL SCRIPTS ~~~~~~~~~~~~~~~~
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# ~~~~~~~~~~~ INSTALL DEPENDENCY SCRIPT ~~~~~~~~~~~~~~
+
+# Install system requirements for MDAI
+# .SILENT: mdai-system-reqs
+.PHONY: mdai-system-reqs
+mdai-system-reqs:
+	@echo "🟢 Start mdai-system-reqs..."
+	@go version || brew list go || brew install go \
+		kubectl version || brew list kubectl || brew install kubectl \
+		npm -v || brew list npm || brew install npm \
+		cdk version || brew list aws-cdk || brew install aws-cdk \
+		docker -v || brew list --cask docker || brew install --cask docker \
+		docker pull otel/opentelemetry-collector:0.95.0 \
+		helm version || brew list helm || brew install helm \
+		kind version || brew list kind || brew install kind
+	@echo "✅ Complete mdai-system-reqs!"
+
+# Add helm chart repos available on system / machine
+# .SILENT: mdai-add-helm-charts
+.PHONY: mdai-add-helm-charts
+mdai-add-helm-charts:
+	@echo "🟢 Start mdai-add-helm-charts..."
+	@helm repo list | grep prometheus-community || helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+	@helm repo list | grep open-telemetry || helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+	@helm repo list | grep mydecisive || helm repo add mydecisive https://decisiveai.github.io/mdai-helm-charts
+	@echo "✅ Complete mdai-add-helm-charts!"
+
+# Install helm charts to cluster
+.SILENT: mdai-install-helm-charts
+.PHONY: mdai-install-helm-charts
+mdai-install-helm-charts:
+	@echo "🟢 Start mdai-install-helm-charts..."
+	helm repo update
+	helm upgrade -f ./templates/prometheus-values.yaml prometheus prometheus-community/prometheus --install --wait
```

Comment:
I fixed a file path here, changing `prometheus_values.yaml` to `prometheus-values.yaml` when I did my tabs fixes. However I think maybe the values in that file actually might break the metrics scraping 😬